src/components/landing-page/landing-page.controller.ts

```diff
@@ -17,14 +17,12 @@ export class LandingPageController implements angular.IController {
     this.ctrl.orderingPanelVisible = false;
 
     this.$scope.$on('open-overlay-panel', (event: any, item: any) => {
-      let kind = _.get(item, 'resource.kind');
-      if (kind === 'ImageStream') {
-        this.ctrl.selectedImageStream = item;
-        this.ctrl.selectedServiceClass = null;
-      } else {
-        this.ctrl.selectedImageStream = null;
-        this.ctrl.selectedServiceClass = item;
+      if (item.resource.kind === 'Template') {
+        this.ctrl.onTemplateSelected({ template: item.resource });
```

Comment:
Where is it defined as a function if no `on-template-selected` attribute is passed in?  -thanks